pkg/odo/cli/describe/component.go

```diff
@@ -197,6 +212,27 @@ func printHumanReadableOutput(cmp api.Component, devfileObj *parser.DevfileObj)
 	if err != nil {
 		return err
 	}
+
+	if len(cmp.Ingresses) != 0 {
+		log.Info("Kubernetes Ingresses:")
+		for _, ing := range cmp.Ingresses {
+			for _, host := range ing.Hosts {
```

Comment:
If there are no hosts (which might happen if an Ingress only has a default backend and no rules), can we still show the ingress name?

Otherwise, we end up with this in the output:
```
$ odo describe component

...
Kubernetes components:
 •  outerloop-deployment
 •  outerloop-service
 •  outerloop-route-ocp
 •  outerloop-ingress-k8s
 •  outerloop-ingress-k8s-no-rule

Kubernetes Ingresses:
```

But the JSON output shows the ingress name:
```json
{
        "ingresses": [
                {
                        "name": "my-java-springboot-app-no-rule",
                        "hosts": null
                }
        ]

---Truncated---
}
```